packages/api-clients/open-api/purposeTemplateApi.yml

```
@@ -0,0 +1,1064 @@
+tags:
+  - name: purposeTemplate
+    description: Implements purpose template process
+    externalDocs:
+      description: Find out more
+      url: "http://swagger.io"
+  - name: health
+    description: Verify service status
+    externalDocs:
+      description: Find out more
+      url: "http://swagger.io"
+paths:
+  /status:
+    get:
+      security: []
+      tags:
+        - health
+      summary: Health status endpoint
+      description: Return ok
+      operationId: getStatus
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+  /purposeTemplates:
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+    get:
+      tags:
+        - purposeTemplate
+      operationId: getPurposeTemplates
+      description: Retrieve Purpose Templates with pagination and filters
+      parameters:
+        - in: query
+          name: purposeTitle
+          description: filter by purpose title
+          schema:
+            type: string
+        - in: query
+          name: creatorIds
+          description: comma separated sequence of creator IDs
+          schema:
+            type: array
+            items:
+              type: string
+              format: uuid
+            default: []
+          explode: false
+        - in: query
+          name: states
+          description: filter by template state
+          schema:
+            type: array
+            items:
+              $ref: "#/components/schemas/PurposeTemplateState"
+            default: []
+          explode: false
+        - in: query
+          name: offset
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 0
+        - in: query
+          name: limit
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 1
+            maximum: 50
+      responses:
+        "200":
+          description: Purpose Templates requested
+          headers:
+            X-Metadata-Version:
+              $ref: "#/components/headers/MetadataVersionHeader"
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/PurposeTemplate"
+        "400":
+          description: Bad Request
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "403":
+          description: Forbidden
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+    post:
+      tags:
+        - purposeTemplate
+      operationId: createPurposeTemplate
+      description: Creates the Purpose Template (Draft state)
```

Minor, let's align the descriptions and summaries: either we use third person to describe the endpoint, so we say `Creates` (the endpoint creates ...) - or we use the second person and say `Create` (you, the caller, create...).
In M2M spec we are not using the `s`, and we are speaking to the caller, I suggest doing the same (double-check for all endpoints)

```suggestion
      description: Create a Purpose Template (Draft state)
```

---

Comment still unaddressed in different endpoints ("Updates", "Deletes", etc.)